src/main/java/frc/robot/commands/AutoPivotElbowCommand.java

```diff
@@ -0,0 +1,53 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+import frc.robot.Constants.PivotConstants;
+import frc.robot.subsystems.ElbowPivotSubsystem;
+
+public class AutoPivotElbowCommand extends CommandBase {
```

Comment:
I'm assuming Jake has mentioned this, but this command (and others like it) can probably be written with PID loops (e.g. using `PIDCommand` or `PIDSubsystem`).